StatBot/Commands/TuneStat.cs

```diff
@@ -22,11 +22,14 @@ public Activity Run(Activity activity)
 
                 Attachment attachment;
                 var actions = new List<CardAction>();
+                EchoBot.ShowedButtons.Clear(); // удаляем из памяти все кнопки
 
                 foreach (var stat in DataModel.Statistics)
                 {
                     if (stat.IsActive)
-                        actions.Add(new StatisticButton(stat.Id, stat.Name,"Subs").Action);
+                    {
+                        actions.Add(new ActionButton(stat.Id, $"{stat.Id}.{stat.Name}", (int)Constants.ActionTypes.ShowSubs, Constants.ShowButtons).Action);
+                    }
                 }
 
                 Attachment a = new Attachment()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'a' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1481)